demos/bootstrap.js

```diff
@@ -0,0 +1,54 @@
+(function() {
+
+// Find the script element
+var scripts = document.getElementsByTagName( "script" );
+var script = scripts[ scripts.length - 1 ];
+
+// Read the modules
+var modules = script.getAttribute( "data-modules" );
+var pathParts = window.location.pathname.split( "/" );
+var effectsAll = [
+	"effect-blind",
+	"effect-bounce",
+	"effect-clip",
+	"effect-drop",
+	"effect-explode",
+	"effect-fade",
+	"effect-fold",
+	"effect-highlight",
+	"effect-puff",
+	"effect-pulsate",
+	"effect-scale",
+	"effect-shake",
+	"effect-size",
+	"effect-slide"
+];
```

Comment:
I'm worried about this getting out of date (though honestly, I don't anticipate more effects). I suppose it will be obvious if this gets out of date though. Either the new effect just won't work in the demos or the demos will throw an error if we delete an effect.